config/routes.js

```diff
@@ -1,221 +1,362 @@
 module.exports.routes = {
-
-    'get /healthcheck'                             : 'PublicController.healthcheck',
```

Comment:
Could you revert this file back to the way it was? There's so many formatting changes its hard to see what has been added. I think the previous formatting was easier to read.